wallet/vspclientloader.go

```diff
@@ -3,6 +3,7 @@ package wallet
 import (
 	"decred.org/dcrwallet/v5/errors"
 	"decred.org/dcrwallet/v5/internal/loggers"
+	"maps"
```

Comment:
also, please put the stdlib imports in their own "section" above (see almost any other source file for examples).